### ISSUE TITLE
How to inherit python packages from system-wide?

### ISSUE THREAD

**Issue by suliveevil** [brunoarine/org-similarity#6]
I have numpy pandas packages installed on system, how to inherit that instead of install another copy to venv? Thank you very much.

**Comment by brunoarine** [comment]
The way I'm using the Python backend makes it a bit hard to use another venv.

One dirty solution would be the following: 

1) Clone the repo, go to its directory.
2) Activate the venv you want to use (or do nothing if you prefer to use your system environment), and install the `orgsimilarity` package: 

```sh
python3 -m pip install .
```

3) Point `org-similarity-python-interpreter` to another virtual environment's executable:

```lisp
(setq org-similarity-python-interpreter "path/to/python/executable/in/another/venv")
```

Let me know if that works for you.

**Comment by suliveevil** [comment]
Seems some of my org files are not using utf-8 https://github.com/brunoarine/org-similarity/issues/7

I will test this after #7 solved.

Thank you so much.

**Comment by brunoarine** [comment]
Duplicate of #8
